subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/FasterIncrementalAndroidBuildsPerformanceTest.groovy

```diff
@@ -72,7 +79,18 @@ class FasterIncrementalAndroidBuildsPerformanceTest extends AbstractCrossBuildPe
         }
     }
 
-    def configureFastIncrementalBuild(GradleBuildExperimentSpec.GradleBuilder builder) {
-        builder.invocation.args(INSTANT_EXECUTION_PROPERTY)
+    static void passChangedFile(GradleBuildExperimentSpec.GradleBuilder builder, IncrementalAndroidTestProject testProject) {
+        builder.invocation.args("-D${VirtualFileSystemServices.VFS_CHANGES_SINCE_LAST_BUILD_PROPERTY}=${testProject.pathToChange}")
+    }
+
+    enum Optimization {
+        INSTANT_EXECUTION("org.gradle.unsafe.instant-execution"),
```

Comment:
Maybe refer to this via `org.gradle.instantexecution.SystemProperties#isEnabled`? (Needs to be made public, too.)

Reply:
Looks like the property is used all over the place without any references. I introduced a constant in `JavaInstantExecutionPerformanceTest`, so at least it isn't duplicated within the performance test project.